package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "silverstripe-subsites",
+  "description": "Builds the javascript for silverstripe subsites extension",
+  "main": "./client/src/boot/index.js",
+  "author": "You",
+  "engines": {
+    "node": ">=10.x"
```

Comment:
probably want to lock this to ^10 like other core packages e.g. https://github.com/silverstripe/silverstripe-asset-admin/blob/1/package.json#L17

Reply:
Probably correct, although this was originally copied from another supported module, things change over time :)